ggml/examples/unity/unity.cpp

```diff
@@ -182,16 +186,20 @@ int main(int argc, char ** argv) {
 
         // Reset the ggml_context
         model.ctx = ctx_from_buffer(encoder_buf);
-        ggml_set_no_alloc(model.ctx, false);
-        ggml_tensor* seqs = ggml_new_tensor_2d(model.ctx, GGML_TYPE_F32, info.frames, info.channels);
         ggml_set_no_alloc(model.ctx, true);
+        GGML_ASSERT(info.samplerate == 16000);
+        GGML_ASSERT(info.channels == 1);
+        // stop at 30s. Ideally we should chunk input audio, but this will prevent most obvious OOM.
```

Comment:
Can we make this 30 second configurable?

Reply:
like this ?

[ef8c4cd](https://github.com/facebookresearch/seamless_communication/pull/294/commits/ef8c4cd0e7bfa14f11de5522ba87b034cf163fe3)